Docker.DotNet/Properties/AssemblyInfo.cs

```diff
@@ -29,5 +29,5 @@
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
 
-[assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.0.0.0")]
\ No newline at end of file
+[assembly: AssemblyVersion("1.16.0.0")]
+[assembly: AssemblyFileVersion("1.16.0.0")]
```

Comment:
Thanks, man....appreciate it.

-----Original Message-----
From: "Ahmet Alp Balkan" <notifications@github.com>
Sent: ‎8/‎20/‎2015 3:27 PM
To: "ahmetalpbalkan/Docker.DotNet" <Docker.DotNet@noreply.github.com>
Cc: "Josh Garverick" <josh.garverick@gmail.com>
Subject: Re: [Docker.DotNet] Adding new NuSpec file with Microsoft.Bcl.Asyncdependency. (#19)

In Docker.DotNet/Properties/AssemblyInfo.cs:
> \ No newline at end of file
> +[assembly: AssemblyVersion("1.16.0.0")]
> +[assembly: AssemblyFileVersion("1.16.0.0")]
@jgarverick ok I guess I'll just merge like this and then take a fix. Thanks. I hope all goes well for your daughter.  
—
Reply to this email directly or view it on GitHub.